source/features/open-all-selected.js

```diff
@@ -1,46 +1,63 @@
 /* eslint-disable no-alert */
 import {h} from 'dom-chef';
 import select from 'select-dom';
-import {isIssueList} from '../libs/page-detect';
+import * as pageDetect from '../libs/page-detect';
 
 const confirmationRequiredCount = 10;
 
-function getUrlFromCheckbox(checkbox) {
-	return checkbox.closest('li').querySelector('.js-navigation-open').href;
+function getUrlFromItem(element) {
+	// Element could be a checkbox or the issue's <li>
+	return element.closest('li').querySelector('.js-navigation-open').href;
 }
 
-function openSelected() {
-	const selected = select.all('[name="issues[]"]:checked');
+function openIssues() {
+	const issues = select.all([
+		'#js-issues-toolbar.triage-mode + div [name="issues[]"]:checked', // Get checked checkboxes
+		'#js-issues-toolbar:not(.triage-mode) + div .js-issue-row' // Or all items
+	]);
+
 	if (
-		selected.length >= confirmationRequiredCount &&
-		!confirm(`This will open ${selected.length} new tabs. Continue?`)
+		issues.length >= confirmationRequiredCount &&
+		!confirm(`This will open ${issues.length} new tabs. Continue?`)
 	) {
 		return;
 	}
 
 	browser.runtime.sendMessage({
-		urls: selected.map(getUrlFromCheckbox),
+		urls: issues.map(getUrlFromItem),
 		action: 'openAllInTabs'
 	});
 }
 
 export default function () {
-	if (!isIssueList()) {
+	if (!pageDetect.isIssueList() || select.all('.js-issue-row').length < 2) {
 		return;
 	}
+	const openAllButtonText = location.pathname.endsWith('/issues') ? `Open all issues` : `Open all PRs`;
```

Comment:
This was removed here https://github.com/sindresorhus/refined-github/pull/1402/commits/692b01b040d30102067f0a908d3ae33bf9f42c29